addons/base_vat/tests/test_validate_ruc.py

```diff
@@ -97,6 +101,12 @@ def test_nif_de(self):
         with self.assertRaises(ValidationError):
             test_partner.write({'vat': "136695978"})
 
+    def test_soap_client_for_vies_loads(self):
+        with patch.object(Document, '_get_xml_document', return_value=etree.Element("root")), \
+             patch.object(Client, 'service', return_value=None):
+            doc = Document(location=None, transport=Transport())
+            new_get_soap_client(doc, 30)
```

Comment:
There should be a comment explaining the purpose of this test as it's really not obvious.